### PR TITLE
Added: Make + Docker

### PR DESCRIPTION
**Pull request by guillaumephd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,15 +155,18 @@ Then try `DawDreamer`:
...
+If you are on Ubuntu, the following commands will install your dependencies:
 
 ```bash
-sudo apt-get -y install llvm
+sudo apt-get -y install ca-certificates
+sudo apt-get -y install build-essential
 sudo apt-get -y install clang
+sudo apt-get -y install pkg-config
+sudo apt-get -y install libboost-all-dev
+sudo apt-get -y install libboost-python-dev
```
Comment (DBraun) [inline README.md:169]: Just want to double check with you that boost is necessary? Maybe it's a JUCE thing?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    freeglut3-dev \
+    libxcomposite-dev \
+    libcurl4-gnutls-dev \
+    git \
+    python3 \
+    python3-dev \
+&& update-ca-certificates \
+&& apt-get clean -y
+
+RUN git clone --recursive https://github.com/guillaumephd/DawDreamer.git
```
Comment (DBraun) [inline Dockerfile:29]: Can we change this to `RUN git clone --recursive https://github.com/DBraun/DawDreamer.git`

**Review by DBraun — CHANGES_REQUESTED** [review]
Thanks so much for this PR. Could you address the two comments I made?

File: README.md (hunk 1 above)
Comment (turian) [reply README.md:169]: I also needed boost to get a renderman Dockerfile for linux: https://github.com/turian/renderman-dexed-linux/blob/main/Dockerfile

**Pull request merged**